hedera-node/src/main/java/com/hedera/services/store/contracts/HederaWorldState.java

```diff
@@ -437,6 +475,14 @@ public void commit() {
 				final var balanceChange = updatedAccount.getBalance().toLong() - entityAccess.getBalance(accountId);
 				entityAccess.adjustBalance(accountId, balanceChange);
 
+				final Map<UInt256, UInt256> updatedStorage = updatedAccount.getUpdatedStorage();
```

Comment:
These changes are already being committed in the call `commitSizeLimitedStorageTo(entityAccess)` on [L456](https://github.com/hashgraph/hedera-services/blob/e03f33ecc55383d8a7cbdab7b2c4423506e8246b/hedera-node/src/main/java/com/hedera/services/store/contracts/HederaWorldState.java#L456).